src/modules/NtpClient.rb

```diff
@@ -1215,23 +1214,47 @@ def read_known_servers
       servers
     end
 
-    def country_server_label(location, country)
+    # Returns a concatenation of given location and country depending on if
+    # them are empty or not.
+    #
+    # ==== Examples
```

Comment:
You should use [@example tag](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#example) here.